### DISCUSSION TITLE
OON pricing with allowed amount as '0'

### DISCUSSION THREAD

**Discussion by ABPhase2** [CMSgov/price-transparency-guide#564] [Q&A] [answered]
CMS does not have any recommendation on whether or not to report any OON pricing with allowed amounts that are $ 0.

In such scenario, do we report or remove the code that has allowed amount = '0' ?

**Comment by shaselton-usds** [comment, marked as answer]
The data around the `billed_charges` would be lost even if the allowed amount is zero. It is still required.

**Reply by bryanbriegel** [reply]
@shaselton-usds  -  the Transparency in Coverage rule notes at: https://www.federalregister.gov/d/2020-24591/p-684  

"The Departments further clarify that plans and issuers are only required to include in the Allowed Amount File those covered items and services furnished by an out-of-network provider for which the plan or issuer **has adjudicated claims and determined it will pay an allowed amount**.  If the plan or issuer has not adjudicated claims and determined it will pay an allowed amount for items or services furnished by an out-of-network provider, the plan or issuer is not required to include those allowed amounts or billed charges in the Allowed Amount File.
